media-gfx/blender/blender-4.0.2-r1.ebuild

```diff
@@ -287,7 +306,6 @@ src_configure() {
 		-DWITH_INPUT_NDOF=$(usex ndof)
 		-DWITH_INTERNATIONAL=$(usex nls)
 		-DWITH_JACK=$(usex jack)
-		-DWITH_LLVM=$(usex osl)
```

Comment:
If you have LLVM 18 installed, the build will fail in a similar manner to https://bugs.gentoo.org/927281. According to the bundled `FindLLVM.cmake` we should be adding something like this here:

```ebuild
		$(usex osl "-DLLVM_ROOT_DIR=$(get_llvm_prefix)")
```

Reply:
I have tested this and it worked fine for me. Got any logs that show otherwise?